core/src/main/java/com/onloupe/core/serialization/PacketDefinitionList.java

```diff
@@ -118,8 +118,6 @@ public void commit() {
 	public void rollback() {
 		if (this.list.size() > this.committedListSize) {
 			int count = this.list.size() - this.committedListSize;
-			// TODO RKELLIHER review
-			// KM: Pretty sure this won't work because it's iterating forward doing a remove, not backwords.
```

Comment:
This seems to be the standard removeRange algorithm in java, not sure why it won't work?

Also, based on previous code stripping, I'm not sure that this method does anything meaningful at this point. Please advise.

Reply:
If you remove item 50, then go to remove item 51 you'll miss the *old* item 51 because it became item 50 once you did the remove (e.g. when you remove 50 all the others after it should move up one position).  So, you have to tail iterate or just do the remove N times.